INFO:werkzeug:[31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:5000
 * Running on http://137.50.168.187:5000
INFO:werkzeug:[33mPress CTRL+C to quit[0m
INFO:werkzeug: * Restarting with stat
WARNING:werkzeug: * Debugger is active!
INFO:werkzeug: * Debugger PIN: 118-440-111
INFO:werkzeug:127.0.0.1 - - [25/Apr/2024 01:01:37] "GET / HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [25/Apr/2024 01:01:37] "[36mGET /static/index.css HTTP/1.1[0m" 304 -
INFO:werkzeug:127.0.0.1 - - [25/Apr/2024 01:01:37] "[36mGET /static/images/UoA_Primary_Logo_RGB.png HTTP/1.1[0m" 304 -
INFO:werkzeug:127.0.0.1 - - [25/Apr/2024 01:01:37] "[36mGET /static/images/favicon.png HTTP/1.1[0m" 304 -
INFO:werkzeug:127.0.0.1 - - [25/Apr/2024 01:01:40] "[36mGET /static/images/favicon.png HTTP/1.1[0m" 304 -
ERROR:root:{'error_number': 38, 'error': "There is a semantic error where unknown predicate 'or' used in precondition of action. Remove 'or' from the precondition of all actions having it."}
ERROR:root:----- RUNNING FOR ERROR ---> CHAT_GPT
ERROR:root:{'error_number': 38, 'error': "There is a semantic error where unknown predicate 'not' used in precondition of action. Remove 'not' from the precondition of all actions having it."}
ERROR:root:----- RUNNING FOR ERROR ---> CHAT_GPT
ERROR:root:{'error_number': 38, 'error': "There is a semantic error where unknown predicate 'not' used in precondition of action. Remove 'not' from the precondition of all actions having it."}
ERROR:root:----- RUNNING FOR ERROR ---> CHAT_GPT
ERROR:root:{'error_number': 38, 'error': "There is a semantic error where unknown predicate 'not' used in precondition of action. Remove 'not' from the precondition of all actions having it."}
ERROR:root:----- RUNNING FOR ERROR ---> CHAT_GPT
ERROR:root:{'error_number': 38, 'error': "There is a semantic error where unknown predicate 'not' used in precondition of action. Remove 'not' from the precondition of all actions having it."}
ERROR:root:----- RUNNING FOR ERROR ---> CHAT_GPT
ERROR:root:{'error_number': 38, 'error': "There is a semantic error where unknown predicate 'not' used in precondition of action. Remove 'not' from the precondition of all actions having it."}
ERROR:root:----- RUNNING FOR ERROR ---> CHAT_GPT
ERROR:root: < --------- Exited Code -------- > 
ERROR:root: < --------- Exited Code -------- > 
ERROR:root: < --------- Exited Code -------- > 
ERROR:root: < --------- Exited Code -------- > 
ERROR:root: < --------- Exited Code -------- > 
ERROR:root: < --------- Exited Code -------- > 
INFO:werkzeug:127.0.0.1 - - [25/Apr/2024 01:03:23] "POST / HTTP/1.1" 200 -
